string filename
uint8 record_rate
---
